packages/app/src/cli/services/app/config/push.ts

```diff
@@ -40,11 +41,20 @@ export async function pushConfig({configuration, configurationPath}: Options) {
       abort(errors)
     }
 
-    if (usesLegacyScopesBehavior(configuration)) {
+    if (app.requestedAccessScopes && usesLegacyScopesBehavior(configuration)) {
```

Comment:
Would we also want to clear if TOML does not specify `scopes` but sets `legacy_scopes_behavior=true`?